insights/parsers/tests/test_abrt_status_bare.py

```diff
@@ -0,0 +1,31 @@
+from insights.parsers import abrt_status_bare
+from insights.parsers.abrt_status_bare import AbrtStatusBare
+from insights.tests import context_wrap
+
+import doctest
+
+OUTPUT = """
+420
+""".strip()
+
+OUTPUT_MULTILINE = """
+1997
+This line will never exist in real output, but it’s ignored.
```

Comment:
@ernestask this is still failing the python 2 CI tests with the error below.  A simple fix would be just change it's to `it is`.
```
E     File "/tmp/workspace/insights-core_PR-2482-PT65KEKQZGHYZVTBGIFN7CVBNRFUXPIMAM53YMM2HJNPXC5R3JTQ/insights/parsers/tests/test_abrt_status_bare.py", line 13

E   SyntaxError: Non-ASCII character '\xe2' in file /tmp/workspace/insights-core_PR-2482-PT65KEKQZGHYZVTBGIFN7CVBNRFUXPIMAM53YMM2HJNPXC5R3JTQ/insights/parsers/tests/test_abrt_status_bare.py on line 14, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details
```

Reply:
Ugh, I added the magic string to the wrong file. Whatever, fine, I’ll just change the text.

Reply:
Did both, specified the encoding for future generations and dropped the quote.